utils/migration_tools/from_wandb/README.md

```diff
@@ -1,64 +1,83 @@
 - A neptune.ai account, `neptune` python library installed, and environment variables set. Read the [docs](https://docs.neptune.ai/setup/installation/) to learn how to set up your installation.
 
-# Instructions
-- Run `wandb_to_neptune.py`
-- Enter the W&B entity name from where you want to export runs, and the Neptune workspace name where you want to import experiments
-- Enter the projects you want to export as comma-separated values, without spaces. Enter "all" to export all projects
-- Run logs will be created in the same folder as `wandb_to_neptune.py`. You can change this in `logging.basicConfig()`
-
-# Metadata mapping from W&B to Neptune
-
-|     Metadata      |             W&B              |                 Neptune                  |
-| :---------------: | :--------------------------: | :--------------------------------------: |
-|   Project name    |       example_project        |       example-project<sup>1</sup>        |
-|    Project URL    |         project.url          |            project.wandb_url             |
-|     Run name      |           run.name           |               run.sys.name               |
-|      Run ID       |            run.id            |    run.sys.custom_run_id<sup>2</sup>     |
-|       Notes       |          run.notes           |           run.sys.description            |
-|       Tags        |           run.tags           |               run.sys.tags               |
-|      Config       |          run.config          |          run.config<sup>3</sup>          |
-|     Job type      |         run.job_type         |            run.wandb.job_type            |
-|     Run path      |           run.path           |              run.wandb.path              |
-|      Run URL      |           run.url            |              run.wandb.url               |
-|   Creation time   |        run.created_at        |           run.wandb.created_at           |
-|    Run summary    |         run.summary          |         run.summary<sup>3</sup>          |
-|    Run metrics    |      run.scan_history()      |      run.<METRIC_NAME><sup>4</sup>       |
-|  System metrics   | run.history(stream="system") | run.monitoring.<METRIC_NAME><sup>5</sup> |
-|    System logs    |          output.log          |          run.monitoring.stdout           |
-|     Notebook      | code/_session_history.ipynb  |          run.source_code.files           |
-| requirements.txt  |       requirements.txt       |          run.source_code.files           |
-| Model checkpoints |    \*.ckpt/\*checkpoint\*    |             run.checkpoints              |
-|    Other files    |         run.files()          |                run.files                 |
+## Instructions
+
+To use the script, follow these steps:
+
+1. Run `wandb_to_neptune.py`
+2. Enter the source W&B entity name. Leave blank to use your default entity
+3 Enter the destination Neptune workspace name. Leave blank to read from the `NEPTUNE_PROJECT` environment variable
```

Comment:
```suggestion
3. Enter your Neptune project name in the form `workspace-name/project-name`. Leave blank to read from the `NEPTUNE_PROJECT` environment variable.
```
It's not just the workspace name, right?

Reply:
It is. Projects are created automatically